### PR TITLE
Move to typescript

### PR DESCRIPTION
**Pull request by tranlehaiquan**
## Description

Move to Typescript

### CODE REVIEW

**Comment by micleners** [comment]
Nice! 👍   This typescript version will be great for folks who don't know Flow like me.

**Comment by tranlehaiquan** [comment]
> Nice! 👍 This typescript version will be great for folks who don't know Flow like me.

Nice to hear ;)

File: src/components/Post/Comments/Comments.tsx [hunk 1]
```diff
@@ -1,14 +1,12 @@
-// @flow strict
-import React from 'react';
-import ReactDisqusComments from 'react-disqus-comments';
+import React from 'react';import ReactDisqusComments from 'react-disqus-comments';
```
Comment (alxshelepenok) [inline src/components/Post/Comments/Comments.tsx:1]: Codacy has a fix for the issue: [Missing whitespace after semicolon.](https://app.codacy.com/gh/alxshelepenok/gatsby-starter-lumen/pullRequest?prid=7771704)
```suggestion
import React from 'react'; import ReactDisqusComments from 'react-disqus-comments';
```

**Comment by alxshelepenok** [comment]
Thanks for the PR. The starter has been updated to the new version.

**Pull request closed without merging**